music21/harmony.py

```diff
@@ -1711,23 +1778,70 @@ def typeAlter(hD):
         for hD in chordStepModifications:
             if hD.modType == 'add':
                 typeAdd(hD)
-            elif hD.modType == 'subtract':
+            elif hD.modType == 'subtract' or hD.modType == 'omit':
                 typeSubtract(hD)
             elif hD.modType == 'alter':
                 typeAlter(hD)
 
         return tuple(pitches)
 
     def _getKindFromShortHand(self, sH):
+        """
+        Tries to identify a chord kind abbreviation in the given shorthand,
```

Comment:
Thank you for the docs here where the existing code didn't meet current style specifications